@k4zm PR ありがとうございます。
Python2.7 でエラーが出てるようです。codecs.open にして試していただけないでしょうか？

```
$ python setup.py test
Traceback (most recent call last):
  File "setup.py", line 18, in <module>
    with open(rst, 'r', encoding='utf8') as f:
TypeError: 'encoding' is an invalid keyword argument for this function
```

--- a/setup.py
+++ b/setup.py
@@ -9,6 +9,7 @@
     :license: BSD, see LICENSE for more details.
 """
 import os
+from codecs import open
 from setuptools import setup, find_packages
 
 app_name = 'doco'

